### ISSUE TITLE
validation set

### ISSUE THREAD

**Issue by best-wei** [swz30/MPRNet#73]
Hello, thanks for your excellent work, I'm confused about the validation set, is it appropriate to use one of the test sets as the validation set? Hope for your reply sincerely, thanks~

**Comment by adityac8** [comment]
Hi

During training, we use center crop of size 256x256 for validation (not on full-size images) to check if the model is converging well. As long as the model is not overfitting on the test set, it should be fine. For example, once our model is trained on GoPro dataset, we directly test it on HIDE, RealBlur-J, and RealBlur-R datasets and obtain state-of-the-art results (indicating strong generalization).
